packages/angular-ivy/package.json

```diff
@@ -37,7 +38,6 @@
     "@angular/platform-browser-dynamic": "~12.2.0",
     "@angular/router": "~12.2.0",
     "ng-packagr": "^12.1.1",
-    "typescript": "~4.3.5",
```

Comment:
This probably works but let's just keep in mind that we previously already used this version instead of 3.8 for Angular.

Reply:
Jup, this is just so we use the version as defined in the root package.json, one less place to keep this in sync!

Reply:
Given that the compiler throws an error for the wrong version in `@sentry/angular` I still wonder why it works here 😅 NG12 should be used [with TS 4.2.3 to 4.4.0](https://angular.io/guide/versions#unsupported-angular-versions). Maybe they decided to remove this compiler error somewhere between NG10 and12 😅 .
Anyway, as long as this builds correctly, we're probably fine